### PR TITLE
Add coveralls badge

### PR DESCRIPTION
**Pull request by hellwolf**

### CODE REVIEW

File: truffle-config.js [hunk 1]
```diff
@@ -27,7 +27,10 @@ require("dotenv").config();
 
 module.exports = {
 
-    plugins: ["truffle-security", "solidity-coverage"],
+    plugins: [
+        //"truffle-security",
```
Comment (pi0neerpat) [inline truffle-config.js]: Whats the reason for removing?

File: .github/workflows/nodejs.yml [hunk 2]
```diff
@@ -12,15 +13,39 @@ jobs:
...
+
+    - name: npm install, build, and test
+      run: |
+        npm ci
+        npm run test-coverage
+
+    - name: Coveralls
+      uses: coverallsapp/github-action@master
+      with:
+        github-token: ${{ secrets.GITHUB_TOKEN }}
```
Comment (pi0neerpat) [inline .github/workflows/nodejs.yml]: How has your experience been with github flows, vs. automated hooks directly from the tools (e.g., going to coveralls website and adding the repo)

File: truffle-config.js (hunk 1 above)
Comment (hellwolf) [reply truffle-config.js]: it was not part of the package.json dependencies, and we do not have any workflow that requiring it neither. One can still install truffle-security globally and un-comment the line here for personal inquiry.

Do you think we should keep it anyways?

Comment (hellwolf) [reply truffle-config.js]: To keep it we need to add it to package.json dependencies.

File: .github/workflows/nodejs.yml (hunk 2 above)
Comment (hellwolf) [reply .github/workflows/nodejs.yml]: it seems quite straightforward, all I did waws:

- enable rtoken-project access to coveralls
- add the coveralls step to our node CI github action
- go to https://shields.io/category/coverage generate a badge
-done

File: truffle-config.js (hunk 1 above)
Comment (pi0neerpat) [reply truffle-config.js]: no, not necessary

File: .github/workflows/nodejs.yml (hunk 2 above)
Comment (pi0neerpat) [reply .github/workflows/nodejs.yml]: I guess the only difference is that it doesn't show pretty messages here, just ugly ones from github 
image

Comment (hellwolf) [reply .github/workflows/nodejs.yml]: okay,  yea, I hope github will keep improving!

**Pull request merged**